### ISSUE TITLE
disable register email

### ISSUE THREAD

**Issue by mama21mama** [friendica/friendica#8795]
The email service does not work on my server.

How do I deactivate for new registration the email?

**Comment by MrPetovan** [comment]
Hi @mama21mama, sending an email during registration is the only way we have to verify the user has access to this email address so we don't offer disabling it.

Please look into fixing your email sending capabilities instead.

**Comment by tobiasd** [comment]
On Mon, 22 Jun 2020 12:47:43 +0000 (UTC)
mama21mama <notifications@github.com> wrote:

> The email service does not work on my server.
>
> How do I deactivate for new registration the email?

There is an addon to use an external SMTP mail server to send the notification and registration mails. Have a look at "PHP mailer SMTP" in the addon list.

**Comment by mama21mama** [comment]
Noooooo Too bad :( You should be able to disable it.

**Comment by mama21mama** [comment]
addon.config.php

config gmail?.

pagina en blanco friendica.

**Comment by mama21mama** [comment]
`Configuration

You can override the default value of the following config keys in your base Friendica install config/addon.config.php file:

'phpmailer' => [
    // smtp (Boolean)
    // Enables SMTP relaying for outbound emails
    'smtp' => false,

    // smtp_server (String)
    // SMTP server host name
    'smtp_server' => 'smtp.example.com',

    // smtp_port (Integer)
    // SMTP server port number
    'smtp_port' => 25,

    // smtp_secure (String)
    // What kind of encryption to use on the SMTP connection.
    // Options: '', 'ssl' or 'tls'.
    'smtp_secure' => '',

    // smtp_port_s (Integer)
    // Secure SMTP server port number
    'smtp_port_s' => 465,

    // smtp_username (String)
    // SMTP server authentication user name
    // Empty string disables authentication
    'smtp_username' => '',

    // smtp_password (String)
    // SMTP server authentication password
    // Empty string disables authentication
    'smtp_password' => '',

    // smtp_from (String)
    // From address used when using the SMTP server
    // Example: no-reply@example.com
    'smtp_from' => '',
],
`

**Comment by MrPetovan** [comment]
Do you need support to use this addon?

**Comment by mama21mama** [comment]
yes please ... let me friendica the blank home. I do not know what I did wrong.

**Comment by MrPetovan** [comment]
Blank page usually means there a PHP syntax error, please check your web server logs for the error. I'm guessing it is because you edited your `config/addon.config.php` incorrectly, would you mind sharing its content without the secrets (passwords, paths, domains, etc...)

**Comment by mama21mama** [comment]
I already shared it above.

**Comment by MrPetovan** [comment]
Okay. If your file was empty before, it should look like this now:

`config/addon.config.php`
```php
<?php
return [
	'phpmailer' => [
		// smtp (Boolean)
		// Enables SMTP relaying for outbound emails
		'smtp' => false,
		
		// smtp_server (String)
		// SMTP server host name
		'smtp_server' => 'smtp.example.com',
		
		// smtp_port (Integer)
		// SMTP server port number
		'smtp_port' => 25,
		
		// smtp_secure (String)
		// What kind of encryption to use on the SMTP connection.
		// Options: '', 'ssl' or 'tls'.
		'smtp_secure' => '',
		
		// smtp_port_s (Integer)
		// Secure SMTP server port number
		'smtp_port_s' => 465,
		
		// smtp_username (String)
		// SMTP server authentication user name
		// Empty string disables authentication
		'smtp_username' => '',
		
		// smtp_password (String)
		// SMTP server authentication password
		// Empty string disables authentication
		'smtp_password' => '',
		
		// smtp_from (String)
		// From address used when using the SMTP server
		// Example: no-reply@example.com
		'smtp_from' => '',
	],
];

```
Of course after this you need to fill in the configuration details if you have an available SMTP server to send emails from.
